drivers/eeprom/eeprom_mchp_xec.c

```diff
@@ -14,6 +14,10 @@
 #include <zephyr/drivers/pinctrl.h>
 
 #include <zephyr/logging/log.h>
+#include <zephyr/pm/device.h>
+#ifdef CONFIG_PM_DEVICE
+#include <zephyr/pm/policy.h>
+#endif
```

Comment:
Please do not use conditional includes.

```suggestion
#include <zephyr/pm/device.h>
#include <zephyr/pm/policy.h>
```

Reply:
Ok, CONFIG_PM_DEVICE wrapper removed.